contracts/modules/factories/LineFactory.sol

```diff
@@ -52,13 +42,10 @@ contract LineFactory is ILineFactory {
         address oracle_ = oracle; // gas savings
         // deploy new modules
         address s = factory.DeploySpigot(address(this), borrower, borrower);
-        address e = factory.DeployEscrow(defaultMinCRatio, oracle_, address(this), borrower);
-        SecuredLine line = new SecuredLine(oracle_, arbiter, borrower, swapTarget, s, e, ttl, defaultRevenueSplit);
+        address e = factory.DeployEscrow(defaultMinCRatio, oracle, address(this), borrower);
+        SecuredLine line = LineFactoryLib.deploySecuredLine(oracle, arbiter, borrower, swapTarget,s, e, ttl, defaultRevenueSplit);
```

Comment:
Ideally we can move the entire contract logic into the library. So moving factory.deployEscrow to library as well.